SwiftMessageBar/SwiftMessageBar.swift

```diff
@@ -71,6 +71,8 @@ public final class SwiftMessageBar {
     
     private var messageWindow: MessageWindow?
     
+    public var tappedHandler : () -> () = {}
```

Comment:
Could you rather make this optional and in this form:

`public var tapHandler: (() -> Void)?`